# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/streams/readable-streams/owning-type-message-port.any.js.ini
# Commit: c34fddbfd51b
# Full Hash: c34fddbfd51b86477ee9637f291228d8e6c5f939
# Author: Alexandra Borovova <aborovova@mozilla.com>
# Date: 2024-11-28 21:54:11
# Regressor Bug: 1933829
# File Overlap Count: 1
# Description:
#   Bug 1933829 - Mark all tests running in shadowrealm-in-serviceworker as intermittently failing or timing out.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D230557
# ==============================================================================

diff -r 2afdda37f63b -r c34fddbfd51b testing/web-platform/meta/streams/readable-streams/owning-type-message-port.any.js.ini
--- a/testing/web-platform/meta/streams/readable-streams/owning-type-message-port.any.js.ini	Thu Nov 28 15:09:12 2024 +0000
+++ b/testing/web-platform/meta/streams/readable-streams/owning-type-message-port.any.js.ini	Thu Nov 28 16:46:27 2024 +0000
@@ -54,13 +54,4 @@
   expected: ERROR
 
 [owning-type-message-port.https.any.shadowrealm-in-serviceworker.html]
-  expected:
-    if (os == "linux") and fission and not debug and (processor == "x86_64") and not asan and tsan: [TIMEOUT, ERROR]
-    if (os == "linux") and fission and not debug and (processor == "x86_64") and not asan and not tsan: ERROR
-    if (os == "linux") and fission and not debug and (processor == "x86_64") and asan: [TIMEOUT, ERROR]
-    if (os == "win") and debug and (processor == "x86"): ERROR
-    if (os == "linux") and fission and debug: [TIMEOUT, ERROR]
-    if (os == "linux") and not fission and not debug: TIMEOUT
-    if (os == "win") and not debug: ERROR
-    if os == "mac": ERROR
-    [ERROR, TIMEOUT]
+  expected: [ERROR, TIMEOUT]